### ISSUE TITLE
Clarify usage of study dependency section on study edit page

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#351]
**TL;DR**: Clarify usage of study dependency section on study edit page based on researcher confusion

**Narrative**
"As an Experimenter, I would like to understand when to save study dependencies vs. other study details, and what happens when I click each save button."

**Acceptance Criteria**
- [ ] Fix link to docs in study dependencies section
- [ ] Add a "reset to latest" button (ideally, that checks whether this changes anything) 
- [ ] Edit formatting to make it clearer that the "save" button with study dependencies is *not* what to click every time you edit the study JSON
- [ ] User testing

**Comment by kimberscott** [comment]
was duplicated by https://github.com/lookit/lookit-api/issues/390 mistakenly
